cmd/cosign/cli/verify/verify_blob.go

```diff
@@ -270,6 +271,11 @@ func (c *VerifyBlobCmd) Exec(ctx context.Context, blobRef string) error {
 		opts = append(opts, static.WithCertChain(certPEM, chainPEM))
 	}
 
+	co.CTLogPubKeys, err = ctl.GetCTLogPubs(ctx)
```

Comment:
Do you also need to check IgnoreSCT here?

Reply:
I had it there originally, but figured the only harm in fetching the keys here would be in failure cases. I can add a check for it however if you want :)

Reply:
Either way is fine with me too. Technically you could gate under whether there's a cert present for all of these, but I think the logic gets complicated

Reply:
What if you're using a custom TUF root that doesn't specify CT keys (because you're running a private instance for example)? This would err out then I think (unless it always falls back to the embedded keys even if you've configured it to use a specific root? I forget)

Reply:
done :)